windows-iotcore/connect-your-device/Bluetooth.md

```diff
@@ -11,14 +11,34 @@ keywords: windows iot, bluetooth, bluetooth support, devices, device portal
 # Bluetooth Support
 Windows 10 IoT Core supports Bluetooth 4.0. A list of supported Bluetooth dongles can be found in the [Hardware Compatibility List](../learn-about-hardware/HardwareCompatList.md).
 
+# About Bluetooth
+There are two different bluetooth technologies that you can choose to implement in your app:
+
+## Classic Bluetooth (RFCOMM)
+Before Bluetooth LE, devices commonly used this protocol to communicate using Bluetooth. This protocol is simple and useful for device-to-device communication without the need of energy savings. Connectivity requires pairing.
+
+## Bluetooth Low-Energy (BLE/LE)
+Bluetooth Low Energy (LE) is a specification that defines protocols for discovery and communication between devices that have an efficient energy usage requirement. For more information including code samples, As per recent builds, a device can connect to to a BLE device without pairing.
```

Comment:
After "As per recent builds..." it seems like the sentence is incomplete.

Reply:
Not sure how I edit an submitted update so can you please make it. Please change the sentence in question to:
As per recent IoT-Core OS updates, a BLE device can connect to another BLE device without pairing.
Thx